src/release/breaking-changes/key-event-migration.md

```diff
@@ -208,10 +211,11 @@ KeyEventResult _handleKeyEvent(KeyEvent _) {
 }
 ```
 
-Be careful to check conditionals: a [`KeyRepeatEvent`][] is also a key down event,
-but it is a different type (it is not a subclass of [`KeyDownEvent`][]), so
-don't assume that `keyEvent is! KeyDownEvent` only allows key up events, because
-both `KeyDownEvent` and `KeyRepeatEvent` need to be checked.
+Be careful to check conditionals:
+a [`KeyRepeatEvent`][] is also a key down event, but
+it is a different type (it is not a subclass of [`KeyDownEvent`][]), so
+don't assume that `keyEvent is! KeyDownEvent` only allows key up events,
+because both `KeyDownEvent` and `KeyRepeatEvent` need to be checked.
```

Comment:
**issue:** Write as shorter sentences per [GSG](https://developers.google.com/style/accessibility#ease-of-reading).

```suggestion
Be careful to check conditionals. 
Though not inheriting from [`KeyDownEvent`][],
a [`KeyRepeatEvent`][] event works like a key down event.
Don't assume that `keyEvent is! KeyDownEvent` only allows key up events.
Check both `KeyDownEvent` and `KeyRepeatEvent` events.
```